can_tools/scrapers/official/IL/il_vaccine.py

```diff
@@ -11,88 +10,52 @@ class IllinoisVaccineCounty(StateDashboard):
-            raise ValueError(msg)
+    variables = {
+        "AdministeredCount": variables.INITIATING_VACCINATIONS_ALL,
+        "PersonsFullyVaccinated": variables.FULLY_VACCINATED_ALL,
+    }
 
-        return res.json()
-
-    def normalize_all(self, data) -> pd.DataFrame:
-        k = "VaccineAdministration"
-        if k not in data:
-            raise ValueError(f"Expected to find {k} in JSON response")
-        new_names = dict(
-            CountyName="location_name",
-            Report_Date="dt",
-        )
-        df = (
-            pd.DataFrame(data[k])
-            .rename(columns=new_names)
-            .assign(dt=lambda x: pd.to_datetime(x["dt"]))
-            .set_index(["location_name", "dt"])
-        )
-
-        # Select certain columns
-        cmus = {
-            "AdministeredCount": CMU(
-                category="total_vaccine_doses_administered",
-                measurement="cumulative",
-                unit="doses",
-            ),
-            "AllocatedDoses": CMU(
-                category="total_vaccine_allocated",
-                measurement="cumulative",
-                unit="doses",
-            ),
-            "PersonsFullyVaccinated": CMU(
-                category="total_vaccine_completed",
-                measurement="cumulative",
-                unit="people",
-            ),
-        }
-
-        # Reorganize so we can add Chicago to Cook county
-        for cmu_key in cmus.keys():
-            df.loc[pd.IndexSlice["Cook", :], cmu_key] = (
-                df.loc[pd.IndexSlice["Cook", :], cmu_key].values
-                + df.loc[pd.IndexSlice["Chicago", :], cmu_key].values
-            )
+    def fetch(self) -> pd.DataFrame:
+        return pd.read_json(self.url)
 
+    def normalize(self, data: pd.DataFrame) -> pd.DataFrame:
         return (
-            df.reset_index()
-            .melt(id_vars=["location_name", "dt"], value_vars=cmus.keys())
-            .dropna()
-            .assign(
-                value=lambda x: pd.to_numeric(x.loc[:, "value"]),
-                vintage=self._retrieve_vintage(),
+            # rename the Chicago rows such that they are summed together with the Cook County entries,
+            # effectively merging the two locations into one (as CHI is within Cook County)
+            data.replace({"Chicago": "Cook"})
+            .groupby(["CountyName", "Report_Date"])
+            .sum()
+            .reset_index()
+            .loc[
+                :,
+                [
+                    "AdministeredCount",
+                    "PersonsFullyVaccinated",
+                    "Report_Date",
+                    "CountyName",
+                ],
+            ]
+            .pipe(
+                self._rename_or_add_date_and_location,
+                location_name_column="CountyName",
+                location_names_to_drop=[
+                    "Illinois",
+                    "Out Of State",
+                    "Unknown",
+                    "Chicago",
```

Comment:
Out of curiosity, Chicago shouldn't exist anymore at this point, right?

Reply:
ah, yeah it will be merged with Cook County, I'll remove this!

Reply:
My mistake, "Illinois" still needed to be removed so I've added just that one location back